bin/run

```diff
@@ -50,12 +51,11 @@ for vimrc_path in "${VIMRC_PATH[@]}"; do
     fi
 done
 
-mkdir -p $TMPFILE_DIR
-touch $TMPFILE
+xclip -selection clipboard -o > $TMPFILE
 
 # Linux
 if [[ $OSTYPE == "linux-gnu" ]]; then
-  chmod o-r $TMPFILE # Make file only readable by you
+  chmod 600 $TMPFILE # Make file only readable by you
```

Comment:
Should this be one of `chmod 0600` or `chmod go-rwx`?

Reply:
I don't understand your question.

Reply:
Your original command removed permission only from `other`. My suggestion was to ensure that only your user be allowed, i.e. to remove permission from others in the group.